openapi/aro_hcp/v1alpha1/openapi.json

```diff
@@ -2294,6 +2294,10 @@
               "$ref": "#/components/schemas/IdentityProvider"
             }
           },
+          "image_registry": {
```

Comment:
My understanding is that this is configurable only for ARO-HCP Clusters and not for other offerings. The API contract definition should reflect that

Reply:
Correct, it is currently implemented for ARO-HCP.

> The API contract definition should reflect that

IIUC, this update should be omitted from `openapi/clusters_mgmt/v1/openapi.json`. Can you please elaborate on how to do this differentiation?

Reply:
I don't think the tooling allows having a different description between the clusters mgmt and the aro-hcp openapi json files as of now (correct me if that's not the case Elad), so it would be a matter of documenting it in a way that clearly signals the constraints based on the cluster type.

Something like:
For non ARO-HCP Clusters, it is readonly and always enabled.

Alternatives to the "non ARO-HCP Clusters" wording:
* For clusters other than ARO-HCP Clusters

or similar. Then in the future when/if the other offerings adapt to leverage this we would remove / relax this documented constraint.